handlers/src/main/java/com/nextdoor/bender/handler/kinesis/KinesisHandler.java

```diff
@@ -70,4 +77,27 @@ public void onException(Exception e) {
   public InternalEventIterator<InternalEvent> getInternalEventIterator() {
     return this.recordIterator;
   }
+
+  @Override
+  public void prepareMetadata() {
+    /**
```

Comment:
Wrong style of comment for this section. Use `/*` intead of `/**` unless you are describing a class or method.